src/PrestaShopBundle/EventListener/ShopConstraintListener.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
```

Comment:
```suggestion
declare(strict_types = 1);

```

Reply:
Ah good call, and the license header is missing in both PHP files as well